lib/kubeclient/common.rb

```diff
@@ -425,6 +425,7 @@ def api
     def load_entities
       @entities = {}
       fetch_entities['resources'].each do |resource|
+        resource = Kubeclient::Common::Compatibility31.compatible_resource(resource)
```

Comment:
Yes, but I checked it inside the method: https://github.com/abonas/kubeclient/pull/214/files#diff-f1f5af126b7e6ff600fcc39decd74abfR64 thought it would be nicer to hide it. 
Should I check it here instead?